.github/workflows/memory-check.yml

```
@@ -0,0 +1,157 @@
+name: Memory Check
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)